apps/condo/domains/organization/integrations/sbbol/routes.js

```diff
@@ -20,7 +21,7 @@ class SbbolRoutes {
 
         // nonce: to prevent several callbacks from same request
         // state: to validate user browser on callback
-        const checks = { nonce: generators.nonce(), state: generators.state() }
+        const checks = { nonce: generators.nonce(), state: generators.state(), redirectUrl: get(req, 'query.redirectUrl') }
```

Comment:
You need to check it before the save

Reply:
It should not be in `checks`, because checks are for request verification.

Use following structure:

```
req.session[SBBOL_SESSION_KEY] = { checks, redirectUrl }
```